push-notifications/ios/Plugin/PushNotificationsPlugin.swift

```diff
@@ -0,0 +1,188 @@
+import Foundation
+import Capacitor
+import FirebaseCore
```

Comment:
Why is this being tied to firebase? Firebase may be required on Android but iOS should service-agnostic about where the push notifications are coming from. Adding that dependency has significant implications.